### DISCUSSION TITLE
Plugy mode config

### DISCUSSION THREAD

**Discussion by ForestRock22** [theBGuy/Kolbot-SoloPlay#46] [Q&A] [answered]
Hi @theBGuy 

I have plugy installed and I actived it in developer.js but it doesnt look like its working what am I missing?

**Comment by theBGuy** [comment, marked as answer]
Could you provide a little more information? What's not working for it?

**Reply by ForestRock22** [reply]
Everything I suppose I dont understand how its launched through D2B?
Its running fine when I launch it through the plugy exe but nothign with the bots.

**Reply by theBGuy** [reply]
https://github.com/ChaosMarc/PlugY/issues/56 , here this is how I managed to get it set up

**Reply by ForestRock22** [reply]
> [ChaosMarc/PlugY#56](https://github.com/ChaosMarc/PlugY/issues/56) , here this is how I managed to get it set up

Thank you that worked.
